gneiss/plot/_dendrogram.py

```diff
@@ -0,0 +1,235 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, gneiss development team.
+#
+# Distributed under the terms of the GPLv3 License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+"""Drawing trees.
+
+Draws horizontal trees where the vertical spacing between taxa is
```

Comment:
Its both?  The main point of this to provide a layout algorithm.  However, in order to do so, we need a tree representation.  To handle that, we are subclass the skbio TreeNode object, and just assigning attributes directly to nodes in the tree.